File: publisher.js
var PB = require('pubnub');

pub_key = process.env.PUB_KEY; 
sub_key = process.env.SUB_KEY;
sec_key = process.env.SEC_KEY;

var pubnub = PB({
	publish_key : pub_key,
	subscribe_key : sub_key,
	secret_key : sec_key,
});

var choice = function (arr) {
	return arr[Math.floor(arr.length * Math.random())];
}

var random_text = function () {
	out = []
	while (Math.random() > 0.01) {
		out.push(choice("abcdefghijklmnopqrstuvwxyz"));
	}
	return out;
};

var next = function () {
	pubnub.publish({
		channel : "a",
		message : "node: " + random_text()
	});
}

setInterval(next, 5000);

File: publisher.py
from collections import defaultdict
import re
import json
import random
import time
import os
import Pubnub as PB

PUB_KEY = os.environ["PUB_KEY"]
SUB_KEY = os.environ["SUB_KEY"]
SEC_KEY = os.environ["SEC_KEY"]
CHANNEL_NAME = os.environ["CHANNEL_NAME"]

pubnub = PB.Pubnub(
    publish_key = PUB_KEY,
    subscribe_key = SUB_KEY,
    secret_key = SEC_KEY,
    cipher_key = '',
    ssl_on = False,
)

def random_text():
    "generate a random string with a random length"
    out = []
    while random.random() > 0.03:
        out.append(random.choice("abcefghijklmnopqrstuvwxyz"))
    return "".join(out)

if __name__ == "__main__":
    while True:
        pubnub.publish(
            channel=CHANNEL_NAME,
            message= "python:" + random_text(),
        )
        time.sleep(4)
